modules/kinesis/src/it/scala/com/snowplowanalytics/snowplow/enrich/kinesis/EnrichKinesisSpec.scala

```diff
@@ -46,11 +50,24 @@ class EnrichKinesisSpec extends Specification with AfterAll with CatsIO {
       val nbGood = 100l
       val nbBad = 10l
 
+      def containsEnrichmentsContexts(event: Event): Boolean =
+        List(
+          SchemaKey("com.snowplowanalytics.snowplow", "diagnostic_error", "jsonschema", SchemaVer.Full(1, 0, 0)), // API
+          SchemaKey("com.snowplowanalytics.snowplow", "identify", "jsonschema", SchemaVer.Full(1, 0, 0)), // Javascript
+          SchemaKey("com.snowplowanalytics.snowplow", "geolocation_context", "jsonschema", SchemaVer.Full(1, 1, 0)), // SQL
+          YauaaEnrichment.outputSchema
+        ).forall { schema =>
+          event.derived_contexts.data.map(_.schema).contains(schema)
+        }
+
       val testResources = Containers.enrich[IO](
         configPath = "modules/kinesis/src/it/resources/enrich/enrich-localstack.hocon",
         loggerName = "count",
         needsLocalstack = true
-      ).flatMap(_ => mkResources[IO](nbGood, nbBad, localstackPort = Containers.localStackPort))
+      )
+        .flatMap(_ => Containers.mysql[IO])
+        .flatMap(_ => Containers.http[IO])
```

Comment:
Is the order of resources important here? Start dependencies (like DB, HTTP server) first then enrich or it doesn't matter?

Reply:
The order is not important, as http server and mysql server are very quick to start and will be up for sure before enrich needs them. What would you recommend then? Is there a way to run the `acquire`s in parallel ?

Reply:
I thought about more strict version, so first mysql/http then enrich: 

```scala

for {
        _ <- mysql[IO](...)
        _ <- httpServer[IO](...)
        _ <- enrich[IO](...)
}
```
Then enrich would started after those 2 first (also stopped before), right?

Reply:
Yep I also prefer this version for the eyes.